meta/run.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# In this file we specify how each microservice should boot.
+#
+
+runtime: native
+
+config_set: 
+   keyvaluestore:
+      bootcmd: /node keyvaluestore.js
+      env:
+         PORT: 9000
+         MICRO_MASTER_ENDPOINT: micro-master.default.svc.cluster.local
+         MICRO_DB_ENDPOINT: micro-db.default.svc.cluster.local
+         MICRO_STORAGE_ENDPOINT: micro-storage.default.svc.cluster.local
+   db:
+      bootcmd: /node db.js
+      env:
+         PORT: 9001
+         MICRO_KEYVALUESTORE_ENDPOINT: micro-keyvaluestore.default.svc.cluster.local
```

Comment:
Currently the last `--env` that gets called will persist - meaning that those baked in `/run/db` script will be preferred over those given to Capstan. What we currently CAN do is: we can put `$PORT` in the script e.g.:
```
# /run/db
--env=PORT=$PORT /node db.js
```
```bash
# Run like this:
$ capstan run --execute "--env=PORT=8000 runscript /run/db"
```

The benefit of this is that in `meta/run.yaml` we can *hint* user what variables are expected to be set and even provide a custom mapping:
```yaml
# meta/run.yaml
db:
  bootcmd: /node db.js
  env:
    PORT: $PORT
    MICRO_KEYVALUESTORE_ENDPOINT: $MICRO_KEYVALUESTORE_ENDPOINT
```
But the problem is that we cannot "set default PORT in `/run/db` and allow user to override it with `capstan run --env PORT=4444` command".

Should we implement logic that inverts priority? So that bootloader would set each `--env` from the `/run/db` script ONLY IF it is not set yet?

/cc @justinc1